### PR TITLE
add an example of using AUTO_RANDOM with PRE_SPLIT_REGIONS

### PR DESCRIPTION
**Pull request by pepezzzz**
add pre_split_region information.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: auto-random.md [hunk 1]
```diff
@@ -96,6 +119,7 @@ TiDB 自动分配的 `AUTO_RANDOM(S, R)` 列值共有 64 位：
 - 分片位的内容通过计算当前事务的开始时间的哈希值而得。要使用不同的分片位数量（例如 10），可以在建表时指定 `AUTO_RANDOM(10)`。
 - 自增位的值保存在存储引擎中，按顺序分配，每次分配完值后会自增 1。自增位保证了 `AUTO_RANDOM` 列值全局唯一。当自增位耗尽后，再次自动分配时会报 `Failed to read auto-increment value from storage engine` 的错误。
 - 关于取值范围：最终生成值包含的最大位数 = 分片位 + 自增位。有符号位的列的取值范围是 `[-(2^(R-1))+1, (2^(R-1))-1]`。无符号位的列的取值范围是 `[0, (2^R)-1]`。
+- `AUTO_RANDOM` 可以与 `PRE_SPLIT_REGIONS` 结合使用，建表成功后就开始预均匀切分 2^(PRE_SPLIT_REGIONS) 个 Region。
```
Comment (bb7133) [inline auto-random.md:122]:
```suggestion
- `AUTO_RANDOM` 可以与 [`PRE_SPLIT_REGIONS`](sql-statements/sql-statement-split-region.md##pre_split_regions) 结合使用，建表成功后就开始预均匀切分 2^(PRE_SPLIT_REGIONS) 个 Region。
```
How about adding a link?(I'm not sure if this is the correct markdown syntax)

**Review by Oreoxmt — APPROVED** [review]

File: auto-random.md [hunk 2]
```diff
@@ -96,6 +119,7 @@ TiDB 自动分配的 `AUTO_RANDOM(S, R)` 列值共有 64 位：
 - 分片位的内容通过计算当前事务的开始时间的哈希值而得。要使用不同的分片位数量（例如 10），可以在建表时指定 `AUTO_RANDOM(10)`。
 - 自增位的值保存在存储引擎中，按顺序分配，每次分配完值后会自增 1。自增位保证了 `AUTO_RANDOM` 列值全局唯一。当自增位耗尽后，再次自动分配时会报 `Failed to read auto-increment value from storage engine` 的错误。
 - 关于取值范围：最终生成值包含的最大位数 = 分片位 + 自增位。有符号位的列的取值范围是 `[-(2^(R-1))+1, (2^(R-1))-1]`。无符号位的列的取值范围是 `[0, (2^R)-1]`。
+- `AUTO_RANDOM` 可以与 [`PRE_SPLIT_REGIONS`](/sql-statements/sql-statement-split-region.md#pre_split_regions) 结合使用，用来在建表成功后就开始预均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region。
```
Comment (qiancai) [inline auto-random.md:122]:
```suggestion
- `AUTO_RANDOM` 可以与 [`PRE_SPLIT_REGIONS`](/sql-statements/sql-statement-split-region.md#pre_split_regions) 结合使用，用来在建表成功后就开始将表中的数据预均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**